packages/docs-app/src/components/navHeader.tsx

```diff
@@ -101,8 +102,24 @@ export class NavHeader extends React.PureComponent<NavHeaderProps> {
                 return <MenuItem href={href} intent={intent} key={v} text={v} />;
             });
         return (
-            <Popover content={<Menu className="docs-version-list">{releaseItems}</Menu>} placement="bottom">
-                <Tag interactive={true} minimal={true} round={true} rightIcon="caret-down">
+            <Popover
+                content={
+                    <Menu aria-label="docs version" className="docs-version-list" id={VERSION_MENU_ID}>
+                        {releaseItems}
+                    </Menu>
+                }
+                placement="bottom"
+                popupKind={PopupKind.MENU}
```

Comment:
Do we need to specify `popupKind` here since `"menu"` is the default value?

Reply:
I didn't realize the default was `menu`, can remove.